frontend/src/stories/Toast.stories.mdx

```diff
@@ -0,0 +1,51 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
+
+<Meta title="Components/Toast" />
+
+# Toast
+
+> Example: A success toast 
+
+Toasts are small generally one-line messages that appear to inform the user of the result of an action, or an update to the current view.
+
+## Structure and Options
+
+The toast consists of:
+
+- An optional icon
+- A text
+- A close button
+
+There are four types of toasts, each with a specific icon and colour associated with it:
+
+An **Information** toast is blue and provides the user with helpful information about the current view. 
+
+> Eg: a blue notification toast (as it exists today)
+
+A **Success** toast is green and indicates to the user that an action has been successfully carried out.
+
+> Eg: a green success toast (as it exists today)
+
+A **Warning** toast is orange and indicates that there are potential risks or possibility of error.
+
+> Eg: an orange toast (is this used somewhere?)
+
+An **Error** toast is red and indicates that something did not not go as expected, and that there is a problem.
+
+> Eg: a red toast (as it exists today, when moving an WP in GANTT view is not possible)
+
+##  Behaviour
+
+The toast appears either as result of user action (a work package attribute is modified, for example) or a background action (there are new notifications).
```

Comment:
We are currently using in context toast also in multiple setting pages. They are neither triggered by a user action or a background action but are more like a contextual information message. For example in the LDAP groups or LDAP authentication admin settings.

Also this rises the question to me if the EE banners are also extended toasts? Do we treat them separately?

Reply:
Yep, good point. I'll add this to our "things to discuss tomorrow on our daily" list.